pkg/mimir/mimir.go

```diff
@@ -115,6 +116,7 @@ type Config struct {
 	RuntimeConfig       runtimeconfig.Config                       `yaml:"runtime_config"`
 	MemberlistKV        memberlist.KVConfig                        `yaml:"memberlist"`
 	QueryScheduler      scheduler.Config                           `yaml:"query_scheduler"`
+	UsageReport         usagestats.Config                          `yaml:"usage_reporter"`
```

Comment:
On this line there's:
- UsageReport
- usagestats
- usage_reporter

Can we have a single name for everything? 

My 5 cents: I'd call everything `UsageStats`/`usagestats`/`usage_stats`, and have a `-usage-stats.enabled` CLI flag.

Reply:
Makes a lot of sense. Was inherited by Loki (where there's no consistency), then I mangled it without adding consistency tho 🤦